fhirpipe/extract/graphql.py

```diff
@@ -18,33 +18,37 @@
 """
 
 
-def build_resources_query(selected_sources=None, selected_resources=None, selected_labels=None):
+def build_resources_query(selected_source=None, selected_resources=None, selected_labels=None):
     """ Builds a graphql query fetching all the resources needed.
 
     Note that the .replace("'", '"') is needed because the graphql needs to have
     strings delimited by double quotes.
     """
     source_filter = (
         """source: {
-                name: { in: %s }
+                name: { equals: "%s" }
             }"""
-        % selected_sources
-        if selected_sources
+        % selected_source
+        if selected_source
+        else ""
+    )
+    resource_filter = (
+        """definition: {
+                type: { in: %s }
```

Comment:
you can still access definition.type but not filter on it (but we can make it filterable in the api, we'll have to to it manually)

Reply:
Hum, this is a pain.
Do you think we should stop filtering by name and use only ids? It would make the CLI harder to use but wouldn't change Pyrog's interface

Reply:
No I think it's more important to use the definition's type here, let's add some kind a filter to pyrog in order for this to work.